rsconnect/api.py

```diff
@@ -81,9 +81,10 @@ class RSConnectServer(AbstractRemoteServer):
     instance of the Connect server.
     """
 
-    def __init__(self, url, api_key, insecure=False, ca_data=None):
+    def __init__(self, url, api_key, jwt=None, insecure=False, ca_data=None):
```

Comment:
A good practice is to always add new kwargs to the end of the parameters, because some users might be passing  args in the original order. Adding the jwt kwarg here will break existing behavior.